### ISSUE TITLE
Allow unsecure https connections

### ISSUE THREAD

**Issue by fabiobento512** [fabiobento512/FRequest#22]
A new option should be added so FRequest allows unsecure http requests. This is useful for self signed certificate connections.

**Comment by Dean-NC** [comment]
Version 1.1c already works fine for me accessing my localhost with http and https (self-signed cert created automatically by Visual Studio).
My sites are on localhost using ASP.Net with .Net 5, and https self-signed works fine in both IIS Express web server and Microsoft Kestrel server.

**Comment by Bizarrus** [comment]
By me, a selfsigned-cert breaks up with following error:
![image](https://github.com/fabiobento512/FRequest/assets/1371300/35e7bcad-0fdc-4d99-bdf7-e6a36bbe8562)

The cert was created by `openssl`:
```
openssl req -x509 -sha256 -days 356 -noenc -newkey rsa:2048 -subj "/CN=example1.fruit/O=fruithost/emailAddress=server@example1.fruit" -keyout /ssl/example2.fruit.key -out /ssl/example2.fruit.cert
```

On Webbrowser, the cert will accepted after invalid-authority-acceptance.

**Comment by SalDaniele** [comment]
Is this still currently unsupported? I am encountering the same issue as Bizarrus, this would be very nice to have

**Comment by fabiobento512** [comment]
> Is this still currently unsupported? I am encountering the same issue as Bizarrus, this would be very nice to have

Yes unfortunately. My life has been complicated lately and I have not been able to implement some of these basic features.

I have not tested it, but it may be possible, as workaround to add temporarily your self-signed certificates as trusted in the operating system you are using (if you have permissions to do it), possibly with that you should not see the error anymore on FRequest.

**Comment by SalDaniele** [comment]
Yes I tried this but no luck, curl works fine but the tool still throws ssl complaints. Great tool otherwise, super easy to use / lightweight, thanks!

**Comment by NETSphereSoft** [comment]
May be someone has the possibility to add that feature based on this intro idea:

```cpp
connect(manager, &QNetworkAccessManager::sslErrors,
        this, [](QNetworkReply *reply, const QList<QSslError> &errors) {
            for (const QSslError &error : errors) {
                if (error.error() == QSslError::SelfSignedCertificate) {
                    reply->ignoreSslErrors({error});
                }
            }
        });
```
